tests/blueprints/test_plates.py

```diff
@@ -124,3 +124,17 @@ def test_post_plates_mlwh_update_failure(app, client, samples, mocked_responses)
                     "Failed to update MLWH with COG UK ids. The samples should have been successfully inserted into Sequencescape."
                 ]
             }
+
+
+def test_get_plates_endpoint_successful(app, client, samples, mocked_responses, mlwh_lh_samples):
```

Comment:
Don't think you need to pass in `mlwh_lh_samples` - we're not touching the warehouses here are we?

Reply:
Oh good point.